### PR TITLE
Task 1.3

### PR DESCRIPTION
**Pull request by tdmnew**
- Installed babel and created `babel.config.json` file 
- Moved tasks to `./src` folder and created build script
- Converted all files to ES6 modules

### CODE REVIEW

File: src/task1.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import readline from "readline";
+
+const task1 = () => {
+  const rl = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout,
+    terminal: false,
+  });
+
+  rl.on("line", (line) => {
```
Comment (Dmitry-White) [inline src/task1.js]: Missed this in the previous PR.
Consider extracting handlers into a separate named functions and splitting the call chain into separate `rl.on(...)` pieces. That does help emphasise the event-driven nature of code and in makes it easier and cleaner to extend with other events. Moreover, error handling / closing is usually placed at the bottom of the file / logic - think of if-else statements where "else" part is the last one.

**Review by Dmitry-White — CHANGES_REQUESTED** [review]
Looks cool!

File: src/task1.js (hunk 1 above)
Comment (tdmnew) [reply src/task1.js]: Cool, how is this Dmitry?

```
  rl.on("line", reverseString);
  rl.on("error", handleError);
  rl.on("close", handleClose);
```

**Review by Dmitry-White — APPROVED** [review]
Look great!

**Pull request merged**